Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+[!!LIST (Authorizations)](../UserInterface/01a_ListAuthorizations.md)
+
+
+# Monitor authorizations
+
+An authorization transaction is the permission that your company is allowed to capture money from the customer via a payment service provider. The processing of the authorization transactions works as follows: 
+-   The payment service provider creates an authorization transaction, when a customer presses the *Pay now* button during the order process.   
+- The payment provider sends then the authorization transaction to the *ActindoCore1 Platform*.    
+- In addition, the shop system creates an ID for this transaction.  
```

Comment:
Vielleicht dieser Punkt in der Erste zusammenfügen? Wenn ich es richtig im Video verstehe, sind es eher 3 Schritte:
- Kunde bestellt im Shop, Shop macht eine Autorhisierung und schickt die Bestellungsinfo mit Authorisierungssnr. an Actindo
- PSP erstellt auch seinerseits Authorisierungsnr. und schickt auch an Actindo
- Actindo kann beide anhand der Authorisation Nr. miteinander verknüpfen

Reply:
Die Nummern kommen von zwei verschiedenen Akteuren